partner_group/models/partner.py

```diff
@@ -0,0 +1,27 @@
+
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see http://www.gnu.org/licenses/.
+#
+##############################################################################
+
+from openerp import models, fields
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    partner_group = fields.Many2one('res.partner', 'Partner Group')
+    in_group = fields.Boolean('Is in Group',
```

Comment:
Esto es incorrecto. Debería ser `Is a group`, y es para decir si un partner es un grupo.